.github/workflows/autoclose-comment.yml

```diff
@@ -39,36 +39,35 @@ jobs:
 
             Thank you for your contribution to scikit-learn and for the effort you have
             put into this PR. This pull request does not yet meet the quality and
-            clarity needed for an effective review. Reviewing time is limited, and our
-            goal is to prioritize well-prepared contributions to keep scikit-learn
-            maintainable. Unless this PR is improved, it will be automatically closed
-            after two weeks.
+            clarity needed for an effective review. Project maintainers have limited
+            time for code reviews, and our goal is to prioritize well-prepared
+            contributions to keep scikit-learn maintainable.
 
 
-            To avoid autoclose and increase the chance of a productive review, please:
+            As the author, you are responsible for driving this PR, which entails
+            presenting its context and your approach, complying with scikit-learn's
+            development standards, fixing test failures, answering questions, addressing
+            maintainer suggestions, and doing additional research if requested. If all
+            that sounds demanding, that's because it is, and if you are a [new
```

Comment:
Not sure about how this phrase comes off: "if all that sounds demanding, that's because it is,"